src/components/Hero/styles.js

```diff
@@ -80,7 +85,46 @@ const styles = theme => ({
     color: '#FF9015',
     textAlign: 'center',
     fontFamily: 'Oswald, Arial, sans-serif',
-  }
+  },
+
+  imageHolder: {
+    position: 'absolute',
+    bottom : '-3px',
```

Comment:
There is probably a more straightforward way to solve this issue like reducing padding, or etc.

Reply:
Made a note of this.